pkgs/build-support/docker/default.nix

```
@@ -673,6 +646,9 @@ rec {
         if [[ -n "$fromImage" ]]; then
           echo "Unpacking base image..."
           tar -C image -xpf "$fromImage"
+
+          cat ./image/manifest.json  | jq -r '.[0].Layers | .[]' > layer-list
```

There are several cases of `cat file |` over the whole file, that could become:

```suggestion
         jq < ./image/manifest.json -r '.[0].Layers  | .[]' > layer-list
```

---

I know that cat is useless, but I find it much more readable.

---

and slower.

---

There is also `jq -r '.[0].Layers | .[]'  ./image/manifest.json > layer-list` :)